tsconfig.json

```diff
@@ -5,7 +5,7 @@
 		"inlineSources": true,
 		"sourceMap": true,
 		"module": "commonjs",
-		"target": "es5",
+		"target": "es6",
```

Comment:
Na na :P Build target is still per default es5, as es6 isn't supported by most browsers still.